en/drafts/originals/understanding-creating-word-embeddings.md

```diff
@@ -24,135 +24,127 @@ doi: XX.XXXXX/phen0000
 
 {% include toc.html %}
 
-# Understanding and Creating Word Embeddings
+## Lesson Goals
+
```

Comment:
I have modified the structure of the lesson by reordering some of the sections. As you can see, it now starts with Lesson Goals. The table of contents now looks like this:
```
## Lesson Goals
## Prerequisites
### System Requirements
### Corpus Size
## Theory: Introducing Concepts
### Word Embeddings
### Word Vectors
### Document-Term Matrices
### word2vec
### Distance in Vector Space
### Vector Math
## Practice: Exploring Nineteenth-Century American Recipes
### Retrieving the Code and Data
### Building your Model's Vocabulary
### Cleaning the Corpus
### Creating your Model
#### Parameters
### Interrogating the Model with Exploratory Queries
### Validating the Model
## Application: Building a Corpus for your own Research
### Important Preliminary Considerations
### Preparing the Texts in your Corpus
## Next Steps
## Acknowledgements
## Endnotes
```
 Please do let me know if this still makes sense, or if you would like to reword some of the headings (in particular, I struggled to find a satisfying way to word `### Interrogating the Model with Exploratory Queries`).